<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditResourceDefaultDialog</class>
 <widget class="QWidget" name="EditResourceDefaultDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0" colspan="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>11</number>
     </property>
     <widget class="QWidget" name="pageFloat">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item row="0" column="0">
        <widget class="QDoubleSpinBox" name="sbFloat"/>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>225</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageVec2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QSpinBox" name="sbVec2x"/>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="sbVec2y"/>
       </item>
       <item row="1" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>225</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageVec3">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QDoubleSpinBox" name="sbVec3x"/>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="sbVec3y"/>
       </item>
       <item row="0" column="2">
        <widget class="QDoubleSpinBox" name="sbVec3z"/>
       </item>
       <item row="1" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>225</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageVec4">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QDoubleSpinBox" name="sbVec4x"/>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="sbVec4y"/>
       </item>
       <item row="0" column="2">
        <widget class="QDoubleSpinBox" name="sbVec4z"/>
       </item>
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="sbVec4w"/>
       </item>
       <item row="1" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>225</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageInt">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QSpinBox" name="sbInt"/>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>59</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageIVec2">
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QSpinBox" name="sbIVec2x"/>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="sbIVec2y"/>
       </item>
       <item row="1" column="0" colspan="2">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>59</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageIVec3">
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QSpinBox" name="sbIVec3x"/>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="sbIVec3y"/>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="sbIVec3z"/>
       </item>
       <item row="1" column="1">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>59</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageIVec4">
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QSpinBox" name="sbIVec4x"/>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="sbIVec4y"/>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="sbIVec4z"/>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="sbIVec4w"/>
       </item>
       <item row="1" column="2">
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>59</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageColor">
      <widget class="QDoubleSpinBox" name="sbCol4r">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="sbCol4g">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="sbCol4b">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>10</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="sbCol4a">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>10</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pageMat3">
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QDoubleSpinBox" name="sbMat300"/>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="sbMat301"/>
       </item>
       <item row="0" column="2">
        <widget class="QDoubleSpinBox" name="sbMat302"/>
       </item>
       <item row="1" column="0">
        <widget class="QDoubleSpinBox" name="sbMat310"/>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="sbMat311"/>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="sbMat312"/>
       </item>
       <item row="2" column="0">
        <widget class="QDoubleSpinBox" name="sbMat320"/>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="sbMat321"/>
       </item>
       <item row="2" column="2">
        <widget class="QDoubleSpinBox" name="sbMat322"/>
       </item>
       <item row="3" column="0" colspan="2">
        <spacer name="verticalSpacer_9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>55</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageMat4">
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <widget class="QDoubleSpinBox" name="sbMat400"/>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="sbMat401"/>
       </item>
       <item row="0" column="2">
        <widget class="QDoubleSpinBox" name="sbMat402"/>
       </item>
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="sbMat403"/>
       </item>
       <item row="1" column="0">
        <widget class="QDoubleSpinBox" name="sbMat410"/>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="sbMat411"/>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="sbMat412"/>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="sbMat413"/>
       </item>
       <item row="2" column="0">
        <widget class="QDoubleSpinBox" name="sbMat420"/>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="sbMat421"/>
       </item>
       <item row="2" column="2">
        <widget class="QDoubleSpinBox" name="sbMat422"/>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleSpinBox" name="sbMat423"/>
       </item>
       <item row="3" column="0">
        <widget class="QDoubleSpinBox" name="sbMat430"/>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="sbMat431"/>
       </item>
       <item row="3" column="2">
        <widget class="QDoubleSpinBox" name="sbMat432"/>
       </item>
       <item row="3" column="3">
        <widget class="QDoubleSpinBox" name="sbMat433"/>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_10">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>147</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageTexture">
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0" colspan="2">
        <widget class="QLineEdit" name="leTextureRef"/>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>207</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pbTexturePick">
         <property name="text">
          <string>Pick</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_11">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>196</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lblName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>198</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblType">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
